ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/AddGraphicsWithRenderer/HeartGraphicGeometryBuilder.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.ArcGISRuntime 100.12
+
+Item {
+
+    property var rtCreate: ArcGISRuntimeEnvironment.createObject
+
+    function createHeartGeometry(centerX, centerY, sideLength, sr) {
```

Comment:
I'm not sure it's worth splitting out a separate QML file in this case since it is really just holding a single function and nothing else. I'd opt for just putting this function in the main qml file of the sample